jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java

```diff
@@ -39,6 +39,13 @@
  */
 public class Utils {
 
+  /**
+   * Private constructor of Utils Class.
+   */
+  private  Utils(){
+    throw new IllegalAccessError("Utility class");
```

Comment:
Why throwing exception instead of leaving it empty?

Reply:
This is a utility class, which means it provides methods and attributes directly. No class should be able to construct an instance of it, even in a method of itself.